[2024-02-26T14:38:09.175+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-03T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509
[2024-02-26T14:38:09.176+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-03T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-03T00:30:00+00:00/task_id=print_current_date_with_jinja'
[2024-02-26T14:38:09.203+0000] {logging_mixin.py:150} INFO - Changing /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-03T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509
[2024-02-26T14:38:09.204+0000] {logging_mixin.py:150} INFO - Failed to change /opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-03T00:30:00+00:00/task_id=print_current_date_with_jinja permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=python_dag_with_jinja/run_id=scheduled__2024-02-03T00:30:00+00:00/task_id=print_current_date_with_jinja'
[2024-02-26T14:38:09.337+0000] {taskinstance.py:1097} INFO - Dependencies not met for <TaskInstance: python_dag_with_jinja.print_current_date_with_jinja scheduled__2024-02-03T00:30:00+00:00 [queued]>, dependency 'Execution Date' FAILED: The execution date is 2024-02-03T00:30:00+00:00 but this is after the task's end date 2024-01-04T00:00:00+00:00.
[2024-02-26T14:38:09.346+0000] {local_task_job_runner.py:154} INFO - Task is not able to be run
